metadata.yaml

```diff
@@ -1,10 +1,12 @@
 homepage: "https://www.snowplowanalytics.com"
-documentation: "https://docs.snowplowanalytics.com/docs"
+documentation: "https://docs.snowplowanalytics.com/docs/forwarding-events-to-destinations/forwarding-events/google-tag-manager-server-side/snowplow-client-for-gtm-ss/"
 versions:
   # Latest version
+  - sha: dacee1e9014574d0b302dff4df5856169faec09e
+    changeNotes: Update Documentation Links (#19)
```

Comment:
Could we also add the `Fix failing enriched event test (#20)` as a change note here?

Reply:
I decided against it, I think we only need to list significant issues in the change notes and I wasn't sure fixing a test really warranted making it into the change notes.